doc/source/whatsnew/v1.2.0.rst

```diff
@@ -297,6 +297,7 @@ I/O
 - :meth:`to_csv` did not support zip compression for binary file object not having a filename (:issue: `35058`)
 - :meth:`to_csv` and :meth:`read_csv` did not honor `compression` and `encoding` for path-like objects that are internally converted to file-like objects (:issue:`35677`, :issue:`26124`, and :issue:`32392`)
 - :meth:`to_picke` and :meth:`read_pickle` did not support compression for file-objects (:issue:`26237`, :issue:`29054`, and :issue:`29570`)
+- :meth:`to_parquet` did not support MultiIndex for columns in parquet format (:issue:`34777`)
```

Comment:
```suggestion
- :meth:`to_parquet` did not support :class:`MultiIndex` for columns in parquet format (:issue:`34777`)
```

Reply:
> Can you also add a test?

Thanks @dsaxton - adding a test for writing MultiIndexes for columns with string column names (test_write_column_multiindex_string).